tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -215,13 +233,14 @@ def testPluginIsActive(self):
 
     self.plugin._thread_for_determining_is_active.run()
 
-    # The plugin later finds that embedding data is available.
-    self.assertTrue(self.plugin.is_active())
+    if USING_REAL_TF:
```

Comment:
I think it's better to skip this whole test.  The `testPluginIsNotActive()` case below already covers the common logic here; the point of this test is to test that the plugin does become active when there is data, but since that doesn't work without real TF, I don't think it's worth running this test without real TF.